src/clr-icons/shapes/essential-shapes.ts

```diff
@@ -902,150 +896,183 @@ export const ClrShapeDotCircle = clrIconSVG(
+  'view-list': ClrShapeViewList,
+  volume: ClrShapeVolume,
+  wand: ClrShapeWand,
   'window-close': ClrShapeWindowClose,
   'window-max': ClrShapeWindowMax,
   'window-min': ClrShapeWindowMin,
   'window-restore': ClrShapeWindowRestore,
+  world: ClrShapeWorld,
+  wrench: ClrShapeWrench,
   'zoom-in': ClrShapeZoomIn,
   'zoom-out': ClrShapeZoomOut,
-  key: ClrShapeKey,
-  library: ClrShapeLibrary,
-  bolt: ClrShapeBolt,
-  wrench: ClrShapeWrench,
-  bullseye: ClrShapeBullseye,
-  target: ClrShapeTarget,
-  flame: ClrShapeFlame,
-  hourglass: ClrShapeHourglass,
-  'no-access': ClrShapeNoAccess,
-  organization: ClrShapeOrganization,
-  balance: ClrShapeBalance,
-  'id-badge': ClrShapeIdBadge,
-  repeat: ClrShapeRepeat,
-  'file-group': ClrShapeFileGroup,
-  paperclip: ClrShapePaperclip,
-  shrink: ClrShapeShrink,
-  'accessibility-1': ClrShapeAccessibility1,
-  'accessibility-2': ClrShapeAccessibility2,
-  'sort-by': ClrShapeSortBy,
-  'collapse-card': ClrShapeCollapseCard,
-  'expand-card': ClrShapeExpandCard,
-  briefcase: ClrShapeBriefcase,
-  'color-picker': ClrShapeColorPicker,
-  'copy-to-clipboard': ClrShapeCopyToClipboard,
-  'filter-2': ClrShapeFilter2,
-  'drag-handle': ClrShapeDragHandle,
-  moon: ClrShapeMoon,
-  sun: ClrShapeSun,
-  wand: ClrShapeWand,
-  clone: ClrShapeClone,
-  details: ClrShapeDetails,
-  'drag-handle-corner': ClrShapeDragHandleCorner,
-  eraser: ClrShapeEraser,
-  landscape: ClrShapeLandscape,
-  paste: ClrShapePaste,
-  portrait: ClrShapePortrait,
-  'file-zip': ClrShapeFileZip,
-  'filter-off': ClrShapeFilterOff,
-  recycle: ClrShapeRecycle,
-  tree: ClrShapeTree,
-  fish: ClrShapeFish,
-  form: ClrShapeForm,
-  fuel: ClrShapeFuel,
-  snowflake: ClrShapeSnowflake,
-  table: ClrShapeTable,
-  'dot-circle': ClrShapeDotCircle,
-  volume: ClrShapeVolume,
-  crosshairs: ClrShapeCrosshairs,
+  /** Alias for: ban */ get cancel(): string {
+    return this['ban'];
+  },
+  /** Alias for: bolt */ get lightning(): string {
+    return this['bolt'];
+  },
+  /** Alias for: bubble-exclamation */ get alert(): string {
+    return this['bubble-exclamation'];
+  },
+  /** Alias for: file */ get document(): string {
+    return this['file'];
+  },
+  /** Alias for: login */ get 'sign-in'(): string {
+    return this['login'];
+  },
+  /** Alias for: logout */ get 'sign-out'(): string {
+    return this['logout'];
+  },
+  /** Alias for: note */ get 'note-edit'(): string {
+    return this['note'];
+  },
+  /** Alias for: organization */ get 'flow-chart'(): string {
+    return this['organization'];
+  },
+  /** Alias for: paperclip */ get attachment(): string {
+    return this['paperclip'];
+  },
+  /** Alias for: pencil */ get edit(): string {
+    return this['pencil'];
+  },
+  /** Alias for: pinboard */ get pinned(): string {
+    return this['pinboard'];
+  },
+  /** Alias for: plus */ get add(): string {
+    return this['plus'];
+  },
+  /** Alias for: resize */ get 'resize-up'(): string {
```

Comment:
As I said before, this is syntactically nicer, and we could leverage TS by doing this way. But we have already tried this in the past. And this got us bad when Webpack/UglifyJS did aggressive minification resulting in runtime errors: 
https://github.com/vmware/clarity/issues/936
https://github.com/vmware/clarity/pull/945

So before getting these changes in, we have to make sure it works as expected with no errors on our end.

Reply:
ok, let me know when you fixed the bugs in your WebPack configuration and I'll rebase this PR to latest.

Reply:
Webpack config is the standard webpack for the Angular-CLI. Its not somehting we have access to change. 

I _think_ what @Shijir means is that your changes need to work with whats already in place for testing, building and publishing).

Reply:
I ran `npm run build:icons` with no issues, it generated the corresponding files (even the minified ones), is there any other script I have to run to make sure this works?